src/libxively/tls/certs/xi_RootCA_list.c

```diff
@@ -8,10 +8,15 @@
 extern "C" {
 #endif
 
-/* generated by create_buffer.py from PEM file xi_RootCA_list.pem
- * ./create_buffer.py --file_name res/trusted_RootCA_certs/xi_RootCA_list.pem \
- *    --array_name xi_RootCA_list --out_path src/libxively/tls/ --no-pretend */
-const unsigned char xi_RootCA_list[5941] = {
+#include "xi_RootCA_list.h"
+
+/* generated by create_buffer.py from a PEM file. Example:
+ * ./create_buffer.py --file_name res/trusted_RootCA_certs/xi_RootCA_list.pem
+ *    --array_name xi_RootCA_list --out_path src/libxively/tls/ --no-pretend
+ *
+ * The buffer is terminated with '\n' for consistency with POSIX filesystem reads
```

Comment:
@atigyi I can remove this comment if you think it's too obvious. It wasn't clear for me at first sight.

Reply:
no, this is great here!